src/pages/questionPage.js

```diff
@@ -27,58 +30,57 @@ export const initQuestionPage = () => {
     const answerElement = createAnswerElement(
       key,
       answerText,
-      currentQuestion.selected
+      createClassListForAnswer(quizData.currentQuestionIndex, key)
     );
     answersListElement.appendChild(answerElement);
 
-    document
-      .getElementById(ANSWERS_OPTION_RADIO_BUTTON_ID + '_' + key)
-      .addEventListener('change', changeOption.bind(null, key));
-
     document
       .getElementById(ANSWERS_OPTION_ID + '_' + key)
       .addEventListener('click', changeOption.bind(null, key));
   }
 
-  if(quizData.currentQuestionIndex < 9) {
-   document
-    .getElementById(NEXT_QUESTION_BUTTON_ID)
-    .addEventListener('click', nextQuestion);
- } else {
-  document.getElementById(NEXT_QUESTION_BUTTON_ID).classList.add("hide")
-  const finishButton = document.createElement('button');
-  finishButton.innerText = "See Results" ;
-  userInterface.appendChild(finishButton)
-
- }
-
+  if (quizData.currentQuestionIndex < 9) {
```

Comment:
Instead of hardcoding the number 9 here, you can use `QUESTION_LIMIT - 1`. Then if you decide to change the number of questions later on, you only have to change it in one place.